app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt

```diff
@@ -361,19 +361,11 @@ object ImageUtils {
         // Calculate your sampleSize based on the requiredWidth and
         // originalWidth
         // For e.g you want the width to stay consistent at 500dp
-        var requiredWidth = 800
+        var requiredWidth = 1920
 
-        if (imageHeight > imageWidth) {
-            requiredWidth = 600
-        }
 
         var sampleSize = ceil((imageWidth.toFloat() / requiredWidth.toFloat()).toDouble()).toInt()
 
-        // If the original image is smaller than required, don't sample
-        if (sampleSize < 1) {
-            sampleSize = 1
-        }
```

Comment:
Why was this removed? From what I understand, this should ensure a small image is not made smaller.

Reply:
If the sampleSize is greater than one, it scales down the image then when it is less than it scales up the image. This logic actually prevents from scaling beyond 800 pixels so it uses as it was captured.
The previous implementation was more focussed on not exceeding 800 pixels rather than setting a minimum resolution. Find attached this image from the android library itself explaining the sample size.
<img width="1440" alt="Screenshot 2022-01-07 at 10 10 56" src="https://user-images.githubusercontent.com/38661872/148523943-532cc0bb-af51-4f62-87ed-d167d0ed153f.png">

Reply:
The documentation actually says if it's less than 1, it doesn't do any scaling. This means the code you removed shouldn't change anything, since it did that internally already. Looks good to me.